forte/data/ontology/top.py

```diff
@@ -1057,13 +1088,18 @@ class Box(Region):
         width: the width of the box, the unit is one image array entry.
     """
 
+    _cy: int
+    _cx: int
+    _height: int
+    _width: int
+
     def __init__(
         self,
         pack: PackType,
         cy: int,
```

Comment:
Will this cause a problem at this [line](https://github.com/asyml/forte/blob/a9998d6acb1ff9b4b01c7e90e8c8bd79fbae1ec3/forte/data/entry_converter.py#L202)? Seems like these arguments are required to instantiate a Box object.

Reply:
Yes you're right. Sorry I missed that.